deploy/autoneg.yaml

```diff
@@ -271,7 +271,16 @@ spec:
           name: https
         securityContext:
           allowPrivilegeEscalation: false
+          capabilities:
```

Comment:
I have not tested this one, because we don't run the kube-rbac-proxy sidecar in our environment